docs/src/content/docs/core-concepts.mdx

```diff
@@ -27,27 +36,27 @@ Themes are authored inside of monorepositories. A theme lives inside a package w
 		- pages/
 		- styles/
 	- index.ts
-	- README.md
 	- package.json
+	- README.md
 </FileTree>
 
-Themes must be authored alongside a playground.
-The playground is responsible for generating types during development and previewing changes to the theme.
+Theme integrations must be authored alongside a playground.
+The playground is responsible for generating types during development and enables previewing changes made to the theme.
 
 <FileTree>
 - playground/
 	- public/
 	- src/
 	- astro.config.mjs
-	- tsconfig.json
 	- package.json
+	- tsconfig.json
 </FileTree>
 
 
-### Theme Config
+### Theme Configuration
 
-Themes have a built-in config that can be configured by a user.
-Theme authors can define a shape for this user configuration with a Zod schema.
+Theme integrations are published with a built-in config file that can be modified by a theme user.
```

Comment:
```suggestion
Theme integrations can be modified by a theme user from inside the Astro config.
```
Theme are configured from inside the Astro config

Reply:
I feel like "the Astro config" is a confusing way to phrase this. Would it be correct to say "the Astro config object" or something?

Reply:
Ya that works, I am not married to my suggestion so whatever you think is best

Reply:
Okay, I changed this section a bit to clarify things. I think the "Astro config object" language is fine if we want to draw a distinction between the file and particular part of the code.